Makefile

```diff
@@ -23,7 +23,7 @@ build:
 	go build ./...
 
 fmt:
-	go fmt ./...
+	gofmt -l -w -s .
```

Comment:
why not bash check_fmt.sh here?

Reply:
we have 2 different use cases - 1) I'm a developer and I want to gofmt all the code in my branch, and 2) I'm travis and I want to verify that a branch is all gofmt compliant and fail if it's not. We currently run the same command for both cases - so travis is actually fixing any non-fmted code and proceeding, hence the need for check_fmt. As a developer, though, I don't want an error message if my code isn't formatted, I just want to fix it.